examples/monitoring/docker/query.py

```diff
@@ -62,7 +62,12 @@ def fizz_buzz(i):
     clipper_conn.start_clipper()
     print("Starting Clipper")
     python_deployer.create_endpoint(
-        clipper_conn, "simple-example", "doubles", feature_sum, num_replicas=2)
+        clipper_conn,
+        "simple-example",
+        "doubles",
+        feature_sum,
+        num_replicas=2,
+        base_image="simonmok/py-rpc")
```

Comment:
Presumably this should be updated to `clipper/py-rpc`?

Reply:
Fixed.